cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -1756,3 +1761,19 @@ def _create_course(test, course_key, course_data):
 def _get_course_id(course_data, key_class=SlashSeparatedCourseKey):
     """Returns the course ID (org/number/run)."""
     return key_class(course_data['org'], course_data['number'], course_data['run'])
+
+
+def _build_other_filter():
```

Comment:
It looks like you need to have at least one asset that has a thumbnail location. This test shows how to set one up:

https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/tests/test_assets.py#L235